core/src/components/range/range.tsx

```diff
@@ -191,6 +191,16 @@ export class Range implements ComponentInterface {
    */
   @Event() ionBlur!: EventEmitter<void>;
 
+  /**
+   * Emitted when the user starts moving the range knob.
+   */
+  @Event() ionChangeStart!: EventEmitter<RangeChangeStartEventDetail>;
+
+  /**
+   * Emitted when the user finishes moving the range knob.
+   */
+  @Event() ionChangeEnd!: EventEmitter<RangeChangeEndEventDetaill>;
```

Comment:
I'm not married to these event names. I know they could be confused with hypothetical `ionWillChange` and `ionDidChange` events, which would imply they fire alongside every `ionChange` event instead of once each. I wanted something more specific than just `ionStart` and `ionEnd`, though. Open to suggestions 🤔